aten/src/ATen/core/jit_type.h

```diff
@@ -946,7 +946,14 @@ struct CAFFE2_API TypeMatchError : public std::exception {
   std::string msg_;
 };
 using TypeEnv = std::unordered_map<std::string, TypePtr>;
```

Comment:
TypeMatchError should be dead code, right?

Reply:
oops yes